packages/bitcore-wallet-client/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "bitcore-wallet-client",
   "description": "Client for bitcore-wallet-service",
   "author": "BitPay Inc",
-  "version": "8.1.0",
```

Comment:
this is the correct version, we incremented this week.

Reply:
thanks, will update all that